src/content/docs/en/guides/integrations-guide/vercel.mdx

```diff
@@ -257,9 +257,14 @@ Note that ISR function requests do not include search params, similar to [reques
 
 By default, an ISR function caches for the duration of your deployment. You can further control caching by setting an expiration time, or by excluding particular routes from caching entirely.
 
+:::caution
```

Comment:
Hi, thanks for your first docs PR!

Just a heads up that we typically only use cautions for dangerous situations (your project won't build, you might lose data) and not just when describing "how things work."

Feel free to include the paragraph above and rework both together entirely so that this reads more like an explanation of how caching works. That's what we normally do when we want to add more information or context! This doesn't have to be a "patch a note on existing text" situation. If we didn't explain it well, or completely, then our explanation should change!